uber/templates/base.html

```diff
@@ -1,32 +1,47 @@
+{% import 'macros.html' as macros %}
 <!DOCTYPE HTML>
 <html>
 <head>
-    <!--[if lte IE 7]>
-        <script>
-            if (!document.documentMode) { //if documentMode exists, this is a later IE in compatibility mode
-                alert("This website requires IE 8 or later.");
-            }
-        </script>
-    <![endif]-->
+    {{ macros.ie7_compatibility_check() }}
     <title>{{ c.EVENT_NAME }} - {% block title %}{% endblock %}</title>
     <link rel="icon" href="../static/images/favicon.png" type="image/x-icon" />
-    <link rel="stylesheet" href="../static/deps/combined.css" />
-    <script type="text/javascript" src="../static/deps/combined.js"></script>
-    <script type="text/javascript">
-        var csrf_token = '{{ c.CSRF_TOKEN }}';
-    </script>
-    <script src="../static/js/common-static.js" type="text/javascript"></script>
+
+    {% block head_styles %}
+        <link rel="stylesheet" href="../static/deps/combined.css" />
+        <link rel="stylesheet" href="../static_views/styles/main.css" />
+        {% if admin_area %}
+            <link rel="stylesheet" type="text/css" href="../static_views/additional-styles.css" />
+        {% else %}
+            <link rel="stylesheet" href="../static/theme/prereg.css">
+            <link rel="stylesheet" href="../static/theme/prereg_extra.css">
+        {% endif %}
+
+         <!-- additional styles -->
+        {% block additional_styles %}
+            {% block page_styles %}{% endblock %}
+        {% endblock %}
+    {% endblock %}
+
+    {% block head_javascript %}
+        <script>
+            var message = '{{ message|e }}';
+        </script>
+
+        <script src="../static/deps/combined.js" type="text/javascript"></script>
+        <script src="../static/js/common-static.js" type="text/javascript"></script>
+
+        {% if 'preregistration' not in c.PAGE_PATH %}
```

Comment:
I made a comment yesterday about not doing this, but, i see now where you are talking about, and agree with it.  Could also leave this as-is and an upcoming PR can do the great c.PAGE_PATH purge.